starknet_devnet/starknet_wrapper.py

```diff
@@ -712,7 +712,7 @@ async def update_pending_block(self, state_update: BlockStateUpdate = None):
 
     async def generate_latest_block(self, block_hash=None) -> StarknetBlock:
         """
-        Generate new block with pending transactions in --blocks-on-demand mode.
+        Generate new block with pending transactions.
```

Comment:
Should we note that it's a block with pending transactions or an empty block if no pending transactions?

Reply:
"Generate new block with pending transactions or empty block." will be ok?